examples/message.js

```diff
@@ -0,0 +1,22 @@
+var phantom = require("../lib");
+var _ph, _page, _outObj;
+
+phantom.create().then(ph => {
+    _ph = ph;
+    return _ph.createPage();
+}).then(page => {
+    _page = page;
+    _ph.onMessage(function(resourceUrl) {
+    	console.log('Node context received resourceUrl: ', resourceUrl);
+    });
+    _page.property('onResourceRequested', function(resource, networkRequest, callbackMessagePrefix) {
+    	console.log(callbackMessagePrefix + resource.url);
```

Comment:
Yes, I would like to provide a `sendMessage()` function to the code running in the `onResourceRequested` callback, but I couldn't figure out where to put that function. In fact.. are there actually 3 JS contexts? Node, Phantom, and the WebKit page itself?